projects/metallb/metallb/helm/schema.json

```diff
@@ -36,7 +36,7 @@
         "name",
```

Comment:
I thought we are using the upstream schema instead of maintaining our own no? Along with setting additionalProperties to true